### PR TITLE
fix image link

### PR DESCRIPTION
**Pull request by ocefpaf**
This is m fault. I missed ~~the `./`~~ that this is a `jekyll` page when I modified #22

### CODE REVIEW

File: _docs/ioos-metadata-profile-v1-2.md [hunk 1]
```diff
@@ -413,6 +413,7 @@ Attributes {
 ## Requirements for IOOS Dataset NDBC/GTS Ingest
 
 In partnership with IOOS, NOAA [NDBC](https://www.ndbc.noaa.gov/) ingests nonfederal IOOS partner data and delivers a subset of those variables (mainly meteorological and physical oceanographic) through the NWS system and on to the WMO [Global Telecommunication System (GTS)](http://www.wmo.int/pages/prog/www/TEM/GTS/index_en.html).  NDBC also publishes all the data they harvest to their web products. 
+
```
Comment (ocefpaf) [inline _docs/ioos-metadata-profile-v1-2.md:416]: The table was not rendering correctly b/c we needed a space here.

File: _docs/ioos-metadata-profile-v1-2.md [hunk 2]
```diff
@@ -433,7 +434,7 @@ In partnership with IOOS, NOAA [NDBC](https://www.ndbc.noaa.gov/) ingests nonfed
 | Eh |   |
 
 NDBC will leverage IOOS RA ERDDAP servers as the central access points for nonfederal IOOS partner data.  This is a departure from the previous data delivery method, where providers and RAs packaged XML files and posted them to an NDBC FTP server. It is critical that RAs work with their regional data providers to ensure their data are available in the RA ERDDAPs, so that NDBC can continue to access those datasets. 
-![Data flow](images/NDBC-data-flow.png?raw=true)
+![Data flow](NDBC-data-flow.png)
```
Comment (ocefpaf) [inline _docs/ioos-metadata-profile-v1-2.md:437]: I forgot that this is a jekyll page so the images are copied to the root of the page.

**Review by kbailey-noaa — APPROVED** [review]
Is there supposed to be a broken image icon below the image?  I see the image so I assume it's fine, just not clear why there's an icon there in the rich text view.

**Review by kbailey-noaa — APPROVED** [review]

**Comment by ocefpaf** [comment]
> Is there supposed to be a broken image icon below the image? I see the image so I assume it's fine, just not clear why there's an icon there in the rich text view.

I completely forgot that this is a `jekyll` page we are not relying GH rendering alone. That means you won't see a broken image in the rich diff, only in the final rendered page. This is a screenshot from the current page:

Screenshot from 2020-05-19 19-17-31

**Comment by mwengren** [comment]
@ocefpaf Yes, it looks like some step in the rendering process for this template must flatten the directory structure, so images get dropped alongside rendered .html pages in the root directory of the repo.  

I suppose the only way to see what link really works is either to: 

* copy someone else's path as an example (I had looked at Bob's Glider DAC pages at one point for this: https://raw.githubusercontent.com/ioos/ioosngdac/gh-pages/_docs/ngdac-netcdf-file-submission-process.md)
* run Jekyll locally to render the page yourself (this counteracts the advantage of editing online in GitHub unfortunately)
* render the site automatically when a pull request is generated?

Is there a way we can do the last one with GitHub Actions for each of our documentation repos?  So even if a page editor like Kathy doesn't see the Jekyll rendered version when saving commits to a branch using GitHub UI, it would show a preview at the point when she submits a PR?  If that's not a difficult thing to do, it might be worthwhile.  

Most likely the documentation theme skeleton project would be the place to do that: https://github.com/ioos/ioos-documentation-jekyll-skeleton, then we could copy it to all the existing doc repos.   I'll file this same comment there.

Will merge these changes now, thanks!

**Pull request merged**